ddl/ddl_api.go

```diff
@@ -230,12 +237,12 @@ func setCharsetCollationFlenDecimal(tp *types.FieldType) error {
 }
 
 func buildColumnAndConstraint(ctx sessionctx.Context, offset int,
-	colDef *ast.ColumnDef) (*table.Column, []*ast.Constraint, error) {
+	colDef *ast.ColumnDef, outPriKeyConstraint *ast.Constraint) (*table.Column, []*ast.Constraint, error) {
```

Comment:
The name of `outPriKeyConstraint ` is not self-explained. Please add a comment for it.